lib/cert.rb

```diff
@@ -13,8 +13,8 @@ module Cert
   class << self
     attr_accessor :config
   end
-  
-  TMP_FOLDER = "/tmp/cert/"
+
+  TMP_FOLDER = ENV["CERT_OUTPUT_PATH"]
```

Comment:
This should be `Cert.config[:output_path]` instead, to also support using `cert --output_path "."`